spark/src/test/scala/org/apache/spark/sql/CometTestBase.scala

```diff
@@ -69,7 +69,7 @@ abstract class CometTestBase
     conf.set("spark.hadoop.fs.file.impl", classOf[DebugFilesystem].getName)
     conf.set("spark.ui.enabled", "false")
     conf.set(SQLConf.SHUFFLE_PARTITIONS, 10) // reduce parallelism in tests
-    conf.set(SQLConf.ANSI_ENABLED.key, "false")
+    conf.set(CometConf.COMET_ANSI_MODE_ENABLED.key, "true")
```

Comment:
What about leaving `COMET_ANSI_MODE_ENABLED` as default value as well? I.e. false for 3.x, true for 4.0

Reply:
Yes, good point